packages/ddp-server/livedata_server.js

```diff
@@ -1224,9 +1224,9 @@ _.extend(Subscription.prototype, {
   onStop: function (callback) {
     var self = this;
     if (self._isDeactivated())
-      callback();
+      callback.call(self);
     else
-      self._stopCallbacks.push(callback);
+      self._stopCallbacks.push(Meteor.bindEnvironment(callback.bind(self)));
```

Comment:
Why don't you just do `callback = Meteor.bindEnvironment(callback.bind(self))` at the beginning?

Not sure why you differentiate between deactivated or not? `Meteor.bindEnvironment` is smart enough to handle if it is not needed.

BTW, `Meteor.bindEnvironment` accepts `this` as the last argument. So you can use that instead of using `callback.bind`.